bloomfilters/matt-keibler/src/HashCalculator.cpp

```diff
@@ -0,0 +1,8 @@
+#include "HashCalculator.h"
+
+Hash HashCalculator::calculate(string word){
+  Hash hash;
+  hash.full_hash = hash_function(word);
```

Comment:
hash_function is a template function from the standard library. On line 20 of HashCalculator.h, I created the hash_function variable to be hash<string>. It is instantiated transparently in the autogenerated constructor of HashCalculator. [std::hash reference](en.cppreference.com/w/cpp/utility/hash)